apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/tour-content.js

```diff
@@ -0,0 +1,98 @@
+/*** THIS MUST BE THE FIRST THING EVALUATED IN THIS SCRIPT *****/
+import './public-path';
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+import welcome from './images/welcome.png';
+import allBlocks from './images/all_blocks.gif';
+import addBlock from './images/add_block.gif';
+import makeBold from './images/make_bold.gif';
+import undo from './images/undo.gif';
+import moveBlock from './images/move_block.gif';
+import finish from './images/finish.png';
+
+/**
+ * This function returns a collection of NUX Tour slide data
+ *
+ * @returns { Array } a collection of <WelcomeTourCard /> props
+ */
+function getTourContent() {
+	return [
+		{
+			heading: __( 'Welcome to Wordpress', 'full-site-editing' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **130** times:
`translate( 'Welcome to WordPress!' )` *ES Score: 8*